src/main/java/org/cactoos/scalar/ItemAt.java

```diff
@@ -194,6 +169,7 @@ public ItemAt(final Iterator<T> iterator, final int position) {
      * @param iterator Iterator
      * @param position Position
      * @param fallback Fallback value
+     * @todo Remove iterator constructor from org.cactoos.scalar.ItemAt
```

Comment:
@mazdack this is not how this work, please read https://github.com/yegor256/pdd and maybe watch https://www.youtube.com/watch?v=nsYGC2aUwfQ